docs/xdebug.md

```diff
@@ -14,7 +14,9 @@ This document describes how to enable Xdebug for local PHP development using the
    `docker-compose down` \
    `docker-compose -f docker-compose.yml -f docker-compose.dev.yml up -d`
    
-   This version of `docker-compose up` uses an override file to modify the image being used by our PHP service. `docker-compose.dev.yml` uses the `esmero-php:development` tagged image that we built in step 1. So we have rebuilt the containers and now you are ready for Part 2.
+   This version of `docker-compose up` uses an override file to modify the image being used by our PHP service. `docker-compose.dev.yml` uses the `esmero-php:development` tagged image that we built in step 1. To stop the containers in the future, run `docker-compose -f docker-compose.yml -f docker-compose.dev.yml down`.
```

Comment:
Marlo. No need to build the image. Its published in docker hub already. Maybe state that Step one is optional?

Reply:
@DiegoPino Ok yes. In this case I need to change other references to `esmero-php:development`. Will do that now.